src/common/types/int128_t.cpp

```diff
@@ -0,0 +1,768 @@
+#include "common/types/int128_t.h"
```

Comment:
My general comment on this file is:
1. Use camel case for function names.
2. Remove any functions that are unused